cypress-io/cypress
# feat(launchpad): Browser launch screen UI

## Purpose

Create a page where users can pick a browser to run tests in.

- Closes [#UNIFY-171](https://cypress-io.atlassian.net/browse/UNIFY-171)

## How to test
Run cypress component test to see this page with a long list of browsers. Run `yarn dev` and navigate to the browser launch screen to see it list the browsers detected on your machine.

## Media

Component in context with real browsers:

![Screen Shot 2021-09-14 at 2 39 27 PM](https://user-images.githubusercontent.com/8340719/133315222-540219a9-88ab-444c-9747-10d8617cf68f.png)

Resizing, hovering, picking options:

https://user-images.githubusercontent.com/8340719/133316255-92f7fff9-7498-4aa9-8cb9-a350263455f7.mov


## Changes

This PR: 

- Adds an `OpenBrowserList` component to render the list inside `OpenBrowser.vue`.
- Makes some changes to Button, Select, Wizard Layout, and Button Bar components that aren't needed any more for the finalized version of this design (single grid of browsers, no advanced selector screen or toggles).

### PR Tasks
